deploy/crds/devopsconsole_v1alpha1_environment_crd.yaml

```diff
@@ -39,4 +39,4 @@ spec:
                   type: string
                 apiGroup:
                   type: string
-  version: v1alpha1
+  version: v1alpha1
```

Comment:
this file name is still `devconsole_...`

Reply:
umm. shows as `...deploy/crds/devopsconsole_v1alpha1_environment_crd.yaml` to me

Reply:
Never mind then. I must be blind:)